lib/ansible/plugins/strategy/free.py

```diff
@@ -152,8 +172,8 @@ def run(self, iterator, play_context):
                                 raise AnsibleError("The '%s' module bypasses the host loop, which is currently not supported in the free strategy "
                                                    "and would instead execute for every host in the inventory list." % task.action, obj=task._ds)
                             else:
-                                display.warning("Using run_once with the free strategy is not currently supported. This task will still be "
-                                                "executed for every host in the inventory list.")
+                                display.debug("Using run_once with the free strategy is not currently supported. This task will still be "
```

Comment:
Was this intentional?  Seems a warning might be better.

Reply:
I agree it should be a warning again. Can't remember if this was in twoerners original work or if I changed that but I can fix it.